integration_tests/run_pyspark_from_build.sh

```diff
@@ -224,7 +224,7 @@ else
     fi
 
     # time zone will be tested; use export TZ=time_zone_name before run this script
-    TZ=${TZ:-UTC}
+    TZ="Asia/Shanghai"
```

Comment:
This cannot stay. We need to have a separate CI job to test for other time zones.

Reply:
Yes, this will be removed before commit. Nightly was prepared in https://github.com/NVIDIA/spark-rapids/issues/9855